packages/fx-core/src/common/spec-parser/manifestUpdater.ts

```diff
@@ -79,15 +79,17 @@ export function generateCommands(
             operationItem.operationId! + ".json"
           );
 
-          const command: Command = {
-            context: ["compose"],
-            type: "query",
-            title: operationItem.summary ?? "",
-            id: operationItem.operationId!,
-            parameters: parameters,
-            apiResponseRenderingTemplate: getRelativePath(manifestPath, adaptiveCardPath),
-          };
-          commands.push(command);
+          if (await fs.pathExists(adaptiveCardPath)) {
```

Comment:
Does this mean if we cannot generate card, then we will not include the command in the manifest?

Reply:
Yes, when generate card failed, it would not generate adaptive card, then will not include the command in the manifest

Reply:
I am thinking about which way would be better:
1. Generate command, but leave the field for card template path empty, so that user won't need to fill in other fields?
2. Keep the command and card path in manifest, and user have to generate a card template with the exact name
3. Skip generating command. User have to add the command manually and fill in each field?

Reply:
I think skip generate command would be better, this will make the manifest workable without needing to update something. We can show warnings to user that some cards are failed to generate, and user can optionally fix it by himself, even he doesn't know how to fix it, the manifest still works for other api.

Reply:
1. As discussed, we plan to generate command with empty path if generate adaptive card failed.
2. Use operationId for the warning data